### DISCUSSION TITLE
Importing PyKDL in blender

### DISCUSSION THREAD

**Discussion by Fa-ediku** [WPI-AIM/ambf#168] [Q&A]
Hi,

I am trying to use Blender to load one of the .blend files in the examples and run the code in that area. In my code, I need to use PyKDL. Currently, I face an error whenever I want to install PyKDL in Blender. It seems that Blender and PyKDL are not using the same python version. As I have seen in your examples that you have used PyKDL too, I wonder if anyone faced the same issue before and if there is a solution for that. Thanks.

**Comment by adnanmunawar** [comment]
Hi, 
We don't use PyKDL in Blender as Blender has its own math library that is sufficient for most of our needs. Secondly, you may get better answers to this question if you post it in a Blender community.
